_tests/browser_tests/jenkins_conf.js

```diff
@@ -0,0 +1,41 @@
+var env = require('./environment.js');
+
+exports.config = {
+  framework: 'jasmine2',
+  specs: ['spec_suites/shared/*.js'],
+  capabilities: {
+    'browserName': 'chrome',
+    'name': 'flapjack-browser-tests'
+  },
+  
+  sauceUser: process.env.SAUCE_USER,
+  sauceKey: process.env.SAUCE_KEY,
+
+  onPrepare: function() {
+    return browser.ignoreSynchronization = true;
```

Comment:
How does this function work? It looks like it'll always return true and never get beyond L:15.